applications/trilinos_application/custom_utilities/parallel_fill_communicator.h

```diff
@@ -24,7 +24,7 @@
 #include "includes/define.h"
 #include "processes/graph_coloring_process.h"
 #include "mpi.h"
-#include "includes/mpi_communicator.h"
+#include "mpi/includes/mpi_communicator.h"
```

Comment:
@pooyan-dadvand and I agreed on not adding the `mpi` folder to the include path by deliberate design. This should make developers aware that they are introducing a dependency on the KratosMPICore when including a file on that route.